examples/README.md

```diff
@@ -7,20 +7,17 @@ name.
 
 ```go
 import (
-	"github.com/google/cel-go/cel"
-	"github.com/google/cel-go/checker/decls"
+    "github.com/google/cel-go/cel"
+    "github.com/google/cel-go/checker/decls"
 )
 
-	d := cel.Declarations(decls.NewIdent("name", decls.String, nil))
-	env, err := cel.NewEnv(d)
+    d := cel.Declarations(decls.NewIdent("name", decls.String, nil))
+    env, err := cel.NewEnv(d)
 
     // Check iss for error in both Parse and Check.
```

Comment:
Now that you're no longer calling `Parse` and `Check` this comment might need to be updated.

Reply:
Done.